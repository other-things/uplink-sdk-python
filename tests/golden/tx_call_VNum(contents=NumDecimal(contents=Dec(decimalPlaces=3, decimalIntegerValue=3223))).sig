r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 71098031543347587287617764221587740195758005559491625208707704289484932109451
