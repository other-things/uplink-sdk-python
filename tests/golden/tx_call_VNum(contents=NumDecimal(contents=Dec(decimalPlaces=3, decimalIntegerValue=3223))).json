{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "args": [
                    {
                        "contents": {
                            "contents": {
                                "decimalIntegerValue": 3223,
                                "decimalPlaces": 3,
                                "tag": "Dec"
                            },
                            "tag": "NumDecimal"
                        },
                        "tag": "VNum"
                    }
                ],
                "method": "get"
            },
            "tag": "Call"
        },
        "tag": "TxContract"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE03MTA5ODAzMTU0MzM0NzU4NzI4NzYxNzc2NDIyMTU4Nzc0MDE5NTc1ODAwNTU1OTQ5MTYyNTIwODcwNzcwNDI4OTQ4NDkzMjEwOTQ1MQ=="
}